modyn/tests/trainer_server/internal/trainer/remote_downsamplers/test_remote_uncertainty_downsampling_strategy.py

```diff
@@ -15,45 +16,45 @@ def get_sampler_config(balance=False):
         "balance": balance,
         "score_metric": "LeastConfidence",
     }
-    return 0, 0, 0, params_from_selector, per_sample_loss_fct, "cpu"
+    return 0, 0, 0, params_from_selector, modyn_config.model_dump(by_alias=True), per_sample_loss_fct, "cpu"
 
 
-def test_init():
-    amds = RemoteUncertaintyDownsamplingStrategy(*get_sampler_config())
+def test_init(dummy_system_config: ModynConfig):
+    amds = RemoteUncertaintyDownsamplingStrategy(*get_sampler_config(dummy_system_config))
 
     assert not amds.requires_coreset_supporting_module
     assert not amds.scores
     assert not amds.index_sampleid_map
     assert not amds.requires_data_label_by_label
 
 
-def test_collect_scores():
-    amds = RemoteUncertaintyDownsamplingStrategy(*get_sampler_config())
+def test_collect_scores(dummy_system_config: ModynConfig):
+    amds = RemoteUncertaintyDownsamplingStrategy(*get_sampler_config(dummy_system_config))
     with torch.inference_mode(mode=(not amds.requires_grad)):
         first_output = torch.randn((4, 5))
         second_output = torch.randn((3, 5))
-        amds.inform_samples([1, 2, 3, 4], first_output, None, None)
+        amds.inform_samples([1, 2, 3, 4], None, first_output, None, None)
```

Comment:
This affects several files, but the expected type is `Union[dict[str, torch.Tensor], torch.Tensor]`. I don't understand why mypy does not complain about `None`. I whether we should rather test with dummy input instead?

Reply:
oh ok. i guess in this particular case it also does not 100% matter. my thought process was that if we give input we check whether anything goes wrong (it won't) but then again it should also go wrong if None is supplied so we can also keep this